support/Pipelines/EccentricityControl/EccentricityControlParams.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import functools
+import logging
+from pathlib import Path
+from typing import Dict, Literal, Optional, Sequence, Tuple, Union
+
+import click
+import h5py
+import numpy as np
+import pandas as pd
+import yaml
+
+from spectre.Visualization.PlotTrajectories import import_A_and_B
+from spectre.Visualization.ReadH5 import to_dataframe
+
+logger = logging.getLogger(__name__)
+
+# Input orbital parameters that can be controlled
+OrbitalParams = Literal[
+    "Omega0",
+    "adot0",
+    "D0",
```

Comment:
Hm these names are actually used a lot, so I tend towards keeping them.

Reply:
Ok I'm fine with this, but we should be aware that if we want to change the names it's better to do it earlier rather than later while we're still building everything. We'll be even more resistant to change in the future once this is being used